tests/test_controller.py

```diff
@@ -445,7 +445,7 @@ def mock_for_library(cls, incoming_library):
         # We also set up some patron web client settings that will
         # be loaded.
         ConfigurationSetting.sitewide(
-            self._db, Configuration.PATRON_WEB_HOSTNAMES).value = json.dumps(["http://sitewide/1234"])
+            self._db, Configuration.PATRON_WEB_HOSTNAMES).value = "http://sitewide/1234"
```

Comment:
Should we be testing a comma separated string value now?

Reply:
Yes, though since I'm suggesting a different separator, I would say there should be a test for a list of URLs. :-)

Reply:
I added an additional test now that tests the parsing with the separator.